docs/docs/api/locate/api-reference.md

```diff
@@ -334,8 +334,16 @@ Here are some sample results with `verbose` set to `true`:
 ```
 
 ### Attribute Descriptions for Responses
+#### Edge
```

Comment:
There are no real docs describing the output of `/locate`, and because the proposed `edge_info.levels` merits some explanation, I figured I'd start.